notifier/tasks.py

```diff
@@ -104,6 +145,69 @@ def _time_slice(minutes, now=None):
     dt_start = dt_end - timedelta(minutes=minutes)
     return (dt_start, dt_end)
 
+
+@celery.task(max_retries=settings.DAILY_TASK_MAX_RETRIES, default_retry_delay=settings.DAILY_TASK_RETRY_DELAY)
+def do_forums_digests_flagged():
+    """
+    Generate and batch send digest emails for each thread with flagged posts.
+    """
+    def get_course_threads():
+        """
+        Process threads from comments service API and group by course_id.
+
+        Returns:
+            dict: key=course_id, value=list of thread URLs with flagged posts
+        """
+        output = {}
+        for thread in get_flagged_threads():
+            course_id = thread.get('course_id')
+            commentable_id = thread.get('commentable_id')
+            comment_thread_id = thread.get('comment_thread_id')
+            if course_id and commentable_id and comment_thread_id:
+                if not output.has_key(course_id):
+                    output[course_id] = []
+                thread_url = '{}/courses/{}/discussion/forum/{}/threads/{}'.format(
```

Comment:
I think the style guide prefers we use named kwargs instead of positional args for string formats.